libs/langchain/langchain/vectorstores/timescalevector.py

```diff
@@ -0,0 +1,889 @@
+"""VectorStore wrapper around a Postgres-TimescaleVector database."""
+from __future__ import annotations
+
+import enum
+import logging
+import uuid
+from datetime import timedelta
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Callable,
+    Dict,
+    Iterable,
+    List,
+    Optional,
+    Tuple,
+    Type,
+    Union,
+)
+
+from langchain.docstore.document import Document
+from langchain.embeddings.base import Embeddings
+from langchain.utils import get_from_dict_or_env
+from langchain.vectorstores.base import VectorStore
+
+if TYPE_CHECKING:
+    from timescale_vector import Predicates
+
+
+class DistanceStrategy(str, enum.Enum):
```

Comment:
believe this is already defined in vectorstores/utils.py, could we use that?

Reply:
Fixed in new commit